python-project-template/.prepare_project.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+echo "Initializing local git repository"
+git init --initial-branch=main >/dev/null
+
+echo "Installing package and runtime dependencies in local environment"
+pip install -e . > /dev/null
+
+echo "Installing developer dependencies in local environment"
+pip install -e .[dev] > /dev/null
```

Comment:
OMG does this work for macs, where they normally have to do `pip install -e '[dev]'`? That's amazing.

Reply:
good catch. In a script surrounding with single quotes is safe so probably we should use them regardless of requirement.

Reply:
Added the single quotes. I know it works on mac, just tested on WSL, so we should be good to go.